django_lightweight_queue/management/commands/queue_deduplicate.py

```diff
@@ -24,7 +31,7 @@ def handle(self, queue, **options):
                 ),
             )
 
-        original_size, new_size = backend.deduplicate(queue)
+        original_size, new_size = backend.deduplicate(queue)  # type: ignore[attr-defined]
```

Comment:
Yeah, I'm not sure about that. I thiiink if it was a `getattr` and we used the result it might be happy with that?

Alternatively it might be that the preferred approach here is either:
- introduce a base method which raises `NotImplementedError` and just catch that, or
- make the type a union around two types -- one with and one without this method

I'm not sure I like any of those approaches really.

I've a feeling that `pyre` rejects cases like this one due to threading concerns, so maybe it's that?

Reply:
76fcdfa fixes this in a nicer way.